### PR TITLE
Add Terrain Exaggeration sample

### PR DESCRIPTION
**Pull request by pgruenler**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Scenes/Terrain Exaggeration/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Terrain Exaggeration
+
+This sample demonstrates how to use terrain exaggeration with a scene.
+
+![](image1.png)
```
Comment (singhgag) [inline arcgis-ios-sdk-samples/Scenes/Terrain Exaggeration/README.md]: Can you resize the image to 320x568? It is pretty big right now.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 2]
```diff
@@ -831,6 +831,14 @@
                         <string>Bristol3DModel</string>
                     </array>
                 </dict>
+                <dict>
+                    <key>displayName</key>
+                    <string>Terrain Exaggeration</string>
+                    <key>storyboardName</key>
+                    <string>TerrainExaggeration</string>
+                    <key>descriptionText</key>
+                    <string>This sample demonstrates how to use terrain exaggeration with a scene</string>
```
Comment (singhgag) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: Should we make this `scene's surface` at the end, to be more specific?

File: arcgis-ios-sdk-samples/Scenes/Terrain Exaggeration/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Terrain Exaggeration
```
Comment (singhgag) [inline arcgis-ios-sdk-samples/Scenes/Terrain Exaggeration/README.md]: 'E' for exaggeration should be lower case

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist [hunk 4]
```diff
@@ -831,6 +831,14 @@
                         <string>Bristol3DModel</string>
                     </array>
                 </dict>
+                <dict>
+                    <key>displayName</key>
+                    <string>Terrain Exaggeration</string>
```
Comment (singhgag) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: 'E' for exaggeration should be lower case

**Comment by singhgag** [comment]
Other than the comments above:
- [ ] Don't forget to rename the folder in the file structure to `Terrain exaggeration` for the readme file to work. Git won't recognize the changes in case as a commit. Talk to @mhdostal, he ended up deleting files and re adding them
- [ ] The UIView in the bottom toolbar has a white background color. Please change that to clear color. And redo the screenshot (320x568)

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist (hunk 2 above)
Comment (pgruenler) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: How about "This sample demonstrates how to use terrain exaggeration on a surface in a scene"

Comment (singhgag) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: 👍

**Comment by pgruenler** [comment]
@singhgag, I made the changes; please take another look.

**Comment by singhgag** [comment]
@pgruenler All those changes look great. Can you do the following changes as well:

1. Remove the `Exaggeration` word from the bottom toolbar. And make it similar to the Map rotation sample [here](https://github.com/Esri/arcgis-runtime-samples-ios/blob/master/arcgis-ios-sdk-samples/Maps/Map%20rotation/image1.png).
2. Add comments to the code.
3. Keep the order of the group in Xcode project as the same in plist (meaning, move the sample group to the last place under Scenes).

Thanks.

**Comment by pgruenler** [comment]
@singhgag, I've made the changes.  Please take another look.

**Comment by singhgag** [comment]
@pgruenler Looks good. Thanks!

**Pull request merged**